### PR TITLE
Port to LLVM 11.0.0 + basic refactoring via Resharper

### PR DESCRIPTION
**Pull request by leskovde**
Code cleanup, setup summary and changes to the newest LLVM / libTooling framework.

### CODE REVIEW

File: test/dot2png.bat [hunk 1]
```diff
@@ -1 +1 @@
-for %%v in (*.dot) do dot -Tpng "%%v" -o "%%~nv.png"
\ No newline at end of file
+for %%v in (*.dot) do "C:\Program Files\Graphviz 2.44.1\bin\dot.exe" -Tpng "%%v" -o "%%~nv.png"
```
Comment (dwat3r) [inline test/dot2png.bat]: Please make this a relative path, like 'dot.exe', for portability.

**Review by dwat3r — CHANGES_REQUESTED** [review]
Hey, thanks for updating my project!

I have a couple of suggestions:
Please use 2 spaces when formatting.
, 4 spaces makes the code too wide.

Please don't delete png files, as they are important showcase what this tool can do.

Could you tell me why do you need this PR to be merged? I'm asking out of curiosity. Updating is nice really.

**Comment by leskovde** [comment]
Hello,
I've reverted all the PNG files I've modified or removed on accident as well as the formatting of the code. 

I'm working on a bachelor thesis. During my research, I was looking for Clang-based slicers and yours is one of the few I've found. I tried to run the code on the latest release of LLVM, however, due to frequent changes to Clang and LibTooling, it just wouldn't compile. 
I've decided that it would be useful for others to update the project to the latest release, rather than download an outdated version of LLVM (which, as I've found out, is not always possible on some systems).

Thanks for making this project available. I'll make sure to reference it in my work :)

Cheers!

**Comment by dwat3r** [comment]
Thank you, this work was my masters thesis, so I'm happy that you've found it useful for yours! :)
Thank you again putting effort into it.
The thing is that it has limited slicing capabilities, so a lot of improvements could be made, but I was lazy to work on it after finishing MSc.

One of the challenges are to make it work with classes, and to slice the whole compilation unit, across functions, excluding standard libraries.

The PR now looks now great, so I'll merge it.

**Review by dwat3r — APPROVED** [review]

**Pull request merged**